SKFlatMaker/script/CRAB3/MakeCrab.py

```diff
@@ -28,7 +28,7 @@
   str_sample = "MC"
   isData = False
   isPrivateMC = False
-if ("Private" in txtfilename) or ("private" in txtfilename):
+if ("Private" in txtfilename) or ("private" in txtfilename) or ("USER" in sys.argv[3]):
```

Comment:
Please explain each agruments if there are more than two (sys.argv[1,2,3]), or better use argparse.. Or I think you can just use your own script, which is what I'm curretly doing

Reply:
Sorry this was not something I meant to do, will modify

Reply:
I meant to do something like
"if "USER" in samplePDs[3] : isPrivateMC = True"
but this kind of line in the middle of 'lines for loop' would make the script complex.
Never mind this change